src/main/java/seedu/eylah/diettracker/logic/commands/HeightCommand.java

```diff
@@ -0,0 +1,50 @@
+package seedu.eylah.diettracker.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.eylah.commons.core.Messages;
+import seedu.eylah.diettracker.logic.commands.exceptions.CommandException;
+import seedu.eylah.diettracker.model.Model;
+import seedu.eylah.diettracker.model.self.Height;
+
+/**
+ * Adds the height of the user in centimeters.
+ */
+public class HeightCommand extends Command {
+
+    public static final String COMMAND_WORD = "height";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": \n"
+            + "Parameters: HEIGHT (must be in centimeters)\n"
+            + "Example: " + COMMAND_WORD + "170";
+
+    public static final String MESSAGE_ADD_HEIGHT_SUCCESS = "Added Height: %1$s";
+
+    private final Height height;
+
+    /**
+     * Constructor for Command that includes input Height.
+     */
+    public HeightCommand(Height height) {
+        this.height = height;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (height.getHt() <= 0) {
```

Comment:
Should we use getHeight instead of getHt()?

Reply:
Yes, agree with @WillySeahh should use getHeight() instead of the abbreviation.

Reply:
ok I agree, I used getHt() because getHeight() returns a String in the original class. I will change the naming conventions to be clearer.